### ISSUE TITLE
No alt text being set for featured images by editors (Accessibility and SEO)

### ISSUE THREAD

**Issue by thetwopct** [cncf/cncf.io#346]
When an image is added by editors, the alt and accessibility text of the image is being ignored.

![20201006104534-Edit Post ‹ Cloud Native Computing Foundation — WordPress](https://user-images.githubusercontent.com/10615884/95179584-1ce84a00-07c1-11eb-9a38-09c695f616ea.jpg)

Look at ways to highlight these to editors to have better data input.

Ideas 
- Highlight input field with heavy red border when empty
- Use Pre-Publish Checklists to highlight missing alt text on featured image and content

**Comment by iamayushdas** [comment]
i want to work on this?

**Comment by cjyabraham** [comment]
This is now addressed
